### PR TITLE
Adds new COC, CLA, Contributing docs and Issue/PR templates

### PR DESCRIPTION
**Pull request by dzenbot**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## New Issues
+
+Before opening a new issue, please consider:
+
+  - Reading [the documentation](https://github.com/slackhq/SlackTextViewController/blob/master/README.md) and [the changelog](https://github.com/slackhq/SlackTextViewController/blob/master/README.md) first.
+  - Searching for any related issues and avoid creating duplicated issues.
+  - Adding details, diagnoses, screenshots or any type of useful information in existing issues, even if they are marked as closed. The team will still review it.
+  - Use the examples [provided in this repository](https://github.com/slackhq/SlackTextViewController/tree/master/examples).
+  - Taking the time to think of a solution and **[open a pull request]**(#new-pull-requests) for either improving the documentation, fixing a bug or suggesting a feature.
```
Comment (johnagan) [inline .github/CONTRIBUTING.md]: These aren't rendering as links. Is this desired or were you looking for `[**open a pull request**](#new-pull-requests)`

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Slack open source code of conduct
+
+
+## Introduction
+Diversity and inclusion make our community strong. We encourage participation from the most varied and diverse backgrounds possible and want to be very clear about where we stand.
+
+Our goal is to maintain a safe, helpful and friendly community for everyone, regardless of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other defining characteristic.
+
+This code and related procedures also apply to unacceptable behavior occurring outside the scope of community activities, in all community venues–online and in-person–as well as in all one-on-one communications, and anywhere such behavior has the potential to adversely affect the safety and well-being of community members.
```
Comment (johnagan) [inline CODE_OF_CONDUCT.md]: ~~in-person–as~~ in-person as

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Expected Behavior
+  * Be welcoming.
+  * Be kind.
+  * Look out for each other.
+
```
Comment (johnagan) [inline CODE_OF_CONDUCT.md]: 👍

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+1. 
+2. 
+3. 
+
+### Expected result:
+
+> e.g. What you expected to happen
+
+### Expected result:
```
Comment (johnagan) [inline .github/ISSUE_TEMPLATE.md]: ~~Expected~~ Actual (?)

**Comment by DEGoodmanWilson** [comment]
LGTM!

**Pull request merged**